### ISSUE TITLE
NgComponentOutlet documentation is broken

### ISSUE THREAD

**Issue by anthony-b-dev** [angular/angular#58936]
At the end of the usage section, the complex example is cut off. 

This line seems to be broken. 

`{@example common/ngComponentOutlet/ts/module.ts region='CompleteExample'}`

**Comment by hawkgs** [comment]
That's right. The code examples are currently not interpolated into the Usage Notes text. It will be handled soon.

Duplicate of https://github.com/angular/angular/issues/56321
